custom_components/tuya_local/devices/trotec_pac-w_heatpump.yaml

```diff
@@ -0,0 +1,80 @@
+name: Air conditioner
+products:
+  - id: 6r9as02gkiaaxelt
+    name: Trotec PAC-W 2600 SH
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: COOL
+              value: cool
+            - dps_val: DRY
+              value: dry
+            - dps_val: FAN
+              value: fan_only
+            - dps_val: HEAT
+              value: heat
+    - id: 2
+      type: integer
+      name: temperature
+      range:
+        min: 16
+        max: 30
+      unit: C
+    - id: 3
+      type: integer
+      name: current_temperature
+    - id: 4
+      type: string
+      name: mode
+      hidden: true
+    - id: 5
+      name: fan_mode
+      type: integer
+      mapping:
+        - dps_val: 1
+          value: low
+        - dps_val: 2
+          value: medium
+        - dps_val: 3
+          value: high
+    - id: 19
+      name: temperature_unit
+      type: string
+    - id: 101
+      name: sleep
+      type: boolean
+    - id: 102
```

Comment:
I'm not sure whether this is an anti-frost mode that could be a switch, or an indicator that could be a binary_sensor (class: cold)

Reply:
I don't know, there is no corresponding button on the remote control or in the app.

Reply:
According "Query Things Data Model" it is readonly
```json
{
	"abilityId": 102,
	"accessMode": "ro",
	"code": "Defrost",
	"description": "0: abtauen\\n1: Auftauen\\n",
	"name": " Defrost",
	"typeSpec": {
		"type": "bool",
		"typeDefaultValue": false
	}
},
```